.circleci/config.yml

```diff
@@ -22,5 +16,12 @@ jobs:
             - run: php composer.phar global show hirak/prestissimo -q || php composer.phar global require --no-interaction --no-progress --optimize-autoloader hirak/prestissimo
             - run: php composer.phar install --optimize-autoloader --no-interaction --no-progress --no-suggest
             - run: php composer.phar info -D | sort
+
+            - save_cache:
+                key: source-v1-{{ .Branch }}-{{ .Revision }}
```

Comment:
don't cache by revision, please, too detailed.

homebrew we can cache by php version, but we have only one currently, so we can cache always
composer's vendor we can cache by php version (also not relevant) and PR target branch (2.12/2.13/master/3.0),
but even if we would not, most of the deps would be cached properly, as 2.12 vs 3.0 has very little deps change.

So, even just adding `restore_cache` would be enough, no need for interpolated cache key

Reply:
Okay, do you know how to speed up `brew update`? It takes usually 5 minutes out of 8 for the build.

Reply:
no, i'm not using mac